File: src/main/java/com/animalfarm/admin/AnimalFarmAdminApplication.java
package com.animalfarm.admin;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;

@SpringBootApplication(exclude = {DataSourceAutoConfiguration.class})
public class AnimalFarmAdminApplication {
    public static void main(String[] args) {
        SpringApplication.run(AnimalFarmAdminApplication.class, args);
    }
}

File: src/main/kotlin/com/animalfarm/admin/AnimalFarmAdminApplication.kt
package com.animalfarm.admin

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration
import org.springframework.boot.runApplication
import org.springframework.data.mongodb.config.EnableMongoAuditing

@SpringBootApplication(exclude = [DataSourceAutoConfiguration::class])
class AnimalFarmAdminApplication

fun main(args: Array<String>) {
    runApplication<AnimalFarmAdminApplication>(*args)
}
